tagnames.py

```
@@ -1,57 +1,57 @@
+#this script creates a locally stored list of tags 
+#To know which TAGS might work in the Model tag search and which do not. 
+#Because there is also stuff on the list from the API that are not working. e.g. comma seperated words 
+#adds only new tags that are not yet present in the tag_names.txt file
+
+
+def fetch_data(url):
+    headers = {"Content-Type": "application/json"}
+    response = requests.get(url, headers=headers)
+    if response.status_code == 200:
+        try:
+            data = response.json()
```

Ruff (0.11.9)

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)